frontend/javascripts/libs/request.js

```diff
@@ -300,13 +300,15 @@ class Request {
               // to a datastore. Then, ping the datastore
               pingMentionedDataStores(text);
 
-              return Promise.reject(json);
+              /* eslint-disable-next-line prefer-promise-reject-errors */
```

Comment:
I'm not sure why we don't use a proper error here (line 307 also uses this pattern). If eslint encourages us to do this, we should probably comply. I opened #5305 as a follow-up.

Reply:
Hmm, I don't know either, thanks for creating the follow-up issue.